src/plugins/cosmos/components/PercentOptionsRow/PercentOptionsRow.tsx

```diff
@@ -0,0 +1,41 @@
+import { Button, ButtonGroup, ButtonGroupProps } from '@chakra-ui/react'
+import { Amount } from 'components/Amount/Amount'
+
+const PERCENT_OPTIONS = [0.25, 0.5, 0.75, 1]
+
+type PercentOptionsRowProps = {
+  onPercentClick: (option: number) => void
+  percent: number | null
+}
+export const PercentOptionsRow = ({
+  percent,
+  onPercentClick,
+  ...styleProps
+}: PercentOptionsRowProps & ButtonGroupProps) => (
+  <ButtonGroup width='100%' bgColor='gray.850' py='5px' borderRadius='12px' {...styleProps}>
+    {PERCENT_OPTIONS.map(option => (
+      <Button
+        isActive={option === percent}
+        key={option}
+        variant='ghost'
+        colorScheme='blue'
```

Comment:
The button group component can take `variant` and spreads it across the underlying children buttons.